guides/manual-installation.md

```diff
@@ -469,13 +469,18 @@ Expected output if Trento web/wanda is ready and the database connection is setu
 {"ready":true}{"database":"pass"}
 ```
 
-## Integrations
+## Single Sign-on Integrations
+
+Trento can be integrated with an identity provider (IDP) that uses diffrent Single sign-on (SSO) protocols like OpenID Connect (OIDC) and Open Authorization 2.0 (OAuth 2).
+
+> [!NOTE]  
+> Trento cannot start with OIDC and OAuth 2 integrations both enabled!
 
 ### OpenID Connect
 
-Trento integrates with an identity provider (IDP) that uses the OpenID Connect (OIDC) protocol to authenticate users accessing the Trento web console. Authorization for specific abilities/permissions is managed by Trento, which means that only basic user information is retrieved from the external IDP.
+Trento integrates with an IDP that uses the OIDC protocol to authenticate users accessing the Trento web console. Authorization for specific abilities/permissions is managed by Trento, which means that only basic user information is retrieved from the external IDP.
 
-#### User Roles and Authentication
+#### User Roles and Authentication for OIDC
```

Comment:
This applies to all SSO integrations. I would extract it out of OIDC as generic chapter.
Otherwise, when we add `SAML` you will need to add other link or similar comment as you did for ouath2 later on